lms/templates/course_modes/track_selection.html

```diff
@@ -157,26 +125,9 @@ <h3 class="py-md-3">
                             <div class="track-selection-choice track-selection-audit ml-md-3">
                                 <p class="float-right m-3 text-uppercase price-display">${_("Free")}</p>
                                 <div class="choice-title"><h4>${_("Access this course")}</h4></div>
-                                <div class="choice-bullets">
-                                    <ul>
-                                        <li>${Text(_("Get temporary access to  {start_bold}non-graded{end_bold} activities, including discussion forums and non-graded assignments")).format(
-                                            start_bold=HTML('<b>'),
-                                            end_bold=HTML('</b>'),
-                                        )}</li>
-                                        <li>${Text(_("Get {start_bold}temporary access{end_bold} to the course material, including videos and readings")).format(
-                                            start_bold=HTML('<b>'),
-                                            end_bold=HTML('</b>'),
-                                        )}</li>
-                                        % if audit_access_deadline:
-                                        <li>${_("Access expires and all progress will be lost on")} ${audit_access_deadline}</li>
-                                        % else:
-                                        <li>${_("Access expires and all progress will be lost")}</li>
-                                        % endif
-                                    </ul>
-                                </div>
+                                <%block name="track_selection_audit_bullets"/>
```

Comment:
I'm not sure I've seen blocks like this before so I want to confirm I'm understanding: it looks like we define those blocks in fbe.html and unfbe.html, and then in track_selection.html we can use the block name to pull them in like little partials- is that right?

Reply:
cool- thanks!